### PR TITLE
Implement `PATCH /user`

### LINKED ISSUE #40: `PATCH /user`
Update the authenticated user

### PR DESCRIPTION
**Pull request by ryumatag**
Closes: #40

### CODE REVIEW

File: sqlx-data.json [hunk 1]
```diff
@@ -90,6 +90,96 @@
...
+            "max_size": 20,
+            "type": "LongLong"
+          }
+        },
+        {
+          "name": "username",
+          "ordinal": 1,
+          "type_info": {
+            "char_set": 224,
+            "flags": {
```
Comment (ryuma017) [inline sqlx-data.json:113]: aaa

**Review by ryumatag — COMMENTED** [review]
aaaaaa

File: Cargo.toml [hunk 2]
```diff
@@ -15,5 +15,4 @@ async-trait = "0.1.58"
 serde = { version = "1.0.147", features = ["derive"] }
 shaku = "0.6.1"
 thiserror = "1.0.37"
-time = { version = "0.3.17", features = ["serde", "formatting", "parsing"] }
-
+time = { version = "0.3.17", features = ["serde-human-readable"] }
```
Comment (ryuma017) [inline Cargo.toml:18]: a

**Review by ryumatag — COMMENTED** [review]
aaa

**Pull request merged**